toolchains/gcc_compatibles.cmake

```diff
@@ -29,7 +29,7 @@ add_compile_options( -fstrict-aliasing -fstrict-enums -fvisibility=hidden -fvisi
 # "Unknown language" error with CMake 3.5.2 if COMPILE_LANGUAGE:C is used.
 # + 'COMPILE_LANGUAGE' isn't supported by VS generators:
 # https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#logical-expressions
-add_compile_options( $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++1z> )
+add_compile_options( $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++14> )
```

Comment:
If we both agree to push for "/std:latest", can you find another way to hack around this? E.g. remove this option afterwards with remove_definitions or override it with another add_compile_options (or a directory or target specific version of it)?